key-manager/src/main/scala/org/bitcoins/keymanager/config/KeyManagerAppConfig.scala

```diff
@@ -0,0 +1,60 @@
+package org.bitcoins.keymanager.config
+
+import java.nio.file.Path
+
+import com.typesafe.config.Config
+import org.bitcoins.core.config.NetworkParameters
+import org.bitcoins.core.util.FutureUtil
+import org.bitcoins.crypto.AesPassword
+import org.bitcoins.db._
+import org.bitcoins.keymanager.WalletStorage
+
+import scala.concurrent.{ExecutionContext, Future}
+
+case class KeyManagerAppConfig(
+    private val directory: Path,
+    private val confs: Config*)(implicit val ec: ExecutionContext)
+    extends AppConfig {
+
+  override def configOverrides: List[Config] = confs.toList
+
+  override type ConfigType = KeyManagerAppConfig
+
+  override def newConfigOfType(
+      configOverrides: Seq[Config]): KeyManagerAppConfig =
+    KeyManagerAppConfig(directory, configOverrides: _*)
+
+  override def moduleName: String = "key-manager"
```

Comment:
I believe we use this for database names? I think the `-` might cause us issues in the future with various things that don't allow `-` in their namespace. I would recommend just doing `keymanager` for simplicity.